src/main/java/com/dpgil/pathlinker/path_linker/internal/Algorithms.java

```diff
@@ -759,8 +764,70 @@ public static CyEdge getEdge(
             }
         }
 
+        // If undirected edges are allowed, then also check for an undirected edge between them
+        // Currently directed edges are favored over undirected edges
+        if (includeUndirected) 
+        {
+			List<CyEdge> undirConnections =
+				network.getConnectingEdgeList(source, target, CyEdge.Type.UNDIRECTED);
+			for (CyEdge edge : undirConnections)
+			{
+				return edge;
+			}
+        	
+        }
+
         return null;
     }
+    
+    
+    /**
+     * Returns 
+     * @param network
+     * 				the network
+     * @param node
+     * 				the node to get the connecting edges
+     * @param edgeType
+     * 				Should be either CyEdge.Type.OUTGOING or CyEdge.Type.INCOMING
+     * @return List<CyEdge> the list of edges (either outgoing or incoming) connected to the given node
```

Comment:
Improve the documentation. Explain what it does based on the value of includeUndirected and why you need this function as a wrapper around the built-in function.